Plugins/Public/alley/PlayerRestrictions.cpp

```diff
@@ -260,16 +268,27 @@ void LoadSettings()
 		ini.close();
 	}
 
-	string scHealingCfgFile = string(szCurDir) + "\\..\\DATA\\EQUIPMENT\\healingrates.cfg";
```

Comment:
have we ensured the live server is ready for this change?

Reply:
Yes, Haste took care of it.